### PR TITLE
LPS-77017

### PR DESCRIPTION
**Pull request by pedroqueiroz94**

### CODE REVIEW

**Comment by pedroqueiroz94** [comment]
ci:test

**Comment by alinedoleron** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_pages_manager.js [hunk 1]
```diff
@@ -1120,7 +1120,7 @@ AUI.add(
 
 						var deletePageButton = instance._getPopover().get('boundingBox').one('.' + CSS_FORM_BUILDER_PAGE_MANAGER_DELETE_PAGE);
 
-						deletePageButton.text(instance._getDeleteButtonString());
+						deletePageButton.html(instance._getDeleteButtonString());
```
[the remark this answers is not in the document]
Comment (alinedoleron) [reply modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_pages_manager.js]: Thanks, @clovisdasilvaneto. I've changed to _getDeleteButtonHTMLString

**Comment by alinedoleron** [comment]
@pedroqueiroz94, I've sent a new PR. https://github.com/pedroqueiroz94/liferay-portal/pull/58

**Comment by alinedoleron** [comment]
ci:close

**Pull request closed without merging**